src/test/java/com/johncnstn/auth/security/DomainUserDetailsServiceTest.java

```diff
@@ -0,0 +1,57 @@
+package com.johncnstn.auth.security;
+
+import com.johncnstn.auth.entity.UserEntity;
+import com.johncnstn.auth.entity.enums.UserRoleEntity;
+import com.johncnstn.auth.repository.UserRepository;
+import com.johncnstn.auth.unit.AbstractUnitTest;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.function.Executable;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.springframework.security.core.userdetails.UsernameNotFoundException;
+
+import java.util.UUID;
+
+import static org.assertj.core.api.SoftAssertions.assertSoftly;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+public class DomainUserDetailsServiceTest extends AbstractUnitTest {
+
+    @Mock
+    private UserRepository userRepository;
+
+    @Test
+    public void testLoadExistingUser() {
+        // GIVEN
+        var userDetailsService = new DomainUserDetailsService(userRepository);
```

Comment:
Don't create beans by yourself, it's unit test, other classes have to be mocked, that's it.

Reply:
Used @InjectMocks for test objects initialization.